<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManipulatorControl</class>
 <widget class="QMainWindow" name="ManipulatorControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ManipulatorControl</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 14pt &quot;Noto Sans&quot;;



</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">

</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="0" column="0" colspan="3">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="CurrentAxisBox">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="title">
          <string>Select Axis</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="0">
             <widget class="QLabel" name="ZValueLabel">
              <property name="text">
               <string>TextLabel</string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="PolarValueLabel">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="AziValueLabel">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QRadioButton" name="ZRadio">
              <property name="text">
               <string>Z [mm]</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="PolarRadio">
              <property name="text">
               <string>Polar [°]</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QRadioButton" name="AziRadio">
              <property name="text">
               <string>Azi [°]</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="styleSheet">
          <string notr="true">QGroupBox{
    font: bold;
    border-radius: 6px;
    margin-top: 6px;
	border: 1px solid darkorange;
}
QGroupBox::title {
    subcontrol-origin: margin;
    left: 7px;
    padding: -10px 5px 0px 5px;
}</string>
         </property>
         <property name="title">
          <string>Continous Movement</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="MoveUpHoldButton">
              <property name="text">
               <string>↑ Up/AntiClock</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="MoveDownHoldButton">
              <property name="text">
               <string>↓ Down/Clock</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QFrame" name="frame_4">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="styleSheet">
          <string notr="true">QGroupBox{
    font: bold;
    border-radius: 6px;
    margin-top: 6px;
	border: 1px solid darkblue;
}
QGroupBox::title {
    subcontrol-origin: margin;
    left: 7px;
    padding: -10px 5px 0px 5px;
}</string>
         </property>
         <property name="title">
          <string>Relative Movements</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="1" column="0">
             <widget class="QPushButton" name="MoveUpRelativeButton">
              <property name="text">
               <string>↑ Up/AntiClock</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="MoveDownRelativeButton">
              <property name="text">
               <string>↓ Down/Clock</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QDoubleSpinBox" name="MoveRelativeBox">
              <property name="minimumSize">
               <size>
                <width>82</width>
                <height>0</height>
               </size>
              </property>
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="maximum">
               <double>200.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QFrame" name="frame_5">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QPushButton" name="StopButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>5</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(248, 112, 0);
font: 36pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>STOP</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="frame_2">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true">QGroupBox{
    font: bold;
    border-radius: 6px;
    margin-top: 6px;
	border: 1px solid darkred;
}
QGroupBox::title {
    subcontrol-origin: margin;
    left: 7px;
    padding: -10px 5px 0px 5px;
}</string>
         </property>
         <property name="title">
          <string>Absolute Movements</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QDoubleSpinBox" name="MoveAbsoluteBox">
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="minimum">
               <double>-33700.000000000000000</double>
              </property>
              <property name="maximum">
               <double>200.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="MoveAbsoluteButton">
            <property name="text">
             <string>Move Absolute</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="GoToInsert">
            <property name="text">
             <string>Go To Insert</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="GoToGamma">
            <property name="text">
             <string>Go To Gamma</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QFrame" name="frame_6">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Communications</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="ComLabel">
              <property name="font">
               <font>
                <family>Noto Sans</family>
                <pointsize>14</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>926</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
